Source/Dafny/Translator.cs

```diff
@@ -7673,7 +7673,7 @@ private class WFOptions
             }
             builder.Add(new Bpl.HavocCmd(me.tok, havocIds));
           }
-          b.Add(Assert(me.tok, Bpl.Expr.False, "missing case in case statement: " + missingCtor.Name));
+          b.Add(Assert(me.tok, Bpl.Expr.False, "missing case in match expression: " + missingCtor.Name));
```

Comment:
Thanks for making this change. Was no test case affected? If not, please add some test case that produces this error message.

Reply:
Oh, the next commit did just that.